spyder/widgets/mixins.py

```diff
@@ -59,6 +59,10 @@ class BaseEditMixin(object):
     _DEFAULT_MAX_HINT_LINES = 20
     _DEFAULT_MAX_HINT_WIDTH = 85
 
+    sig_will_insert_text = None

```

Comment:
Please add a comment above this line explaining why you added these `sig_` attributes here and why they are set to `None`.